UserSessionsMeter.c

```diff
@@ -0,0 +1,157 @@
+/*
+htop - UserSessionsMeter.c
+(C) 2024 htop dev team
+Released under the GNU GPLv2+, see the COPYING file
+in the source distribution for its full text.
+*/
+
+#include "config.h" // IWYU pragma: keep
+
+#include "UserSessionsMeter.h"
+
+#include <dlfcn.h>
+
+
+#include "CRT.h"
+#include "Macros.h"
+#include "Object.h"
+#include "RichString.h"
+#include "XUtils.h"
+#include "generic/utmpx.h"
+
+#if defined(BUILD_STATIC) && defined(HAVE_LIBSYSTEMD)
+#include <systemd/sd-login.h>
+#endif
+
+#ifndef BUILD_STATIC
+#include <dlfcn.h>
+#include "linux/Platform.h"
+#endif
```

Comment:
There should not be a platform-specific reference in the platform-independent code section.